User-Stories/Compliance-Manager-User-Stories/ComplianceManager.md

```diff
@@ -0,0 +1,46 @@
+#Compliance Manager
+This describes the role of a Compliance Manager. This is a role with suitable legal understanding so that he will be able to interpret the legal situation and assess the circumstances accordingly so that he will be able to judge the consequences from obligations respectively support teams / POs in identifying suitable measures to remain compliant. According to OCv2.0 specification, this role may be staffed by external resources.
+
+##Responsibilities, tasks and role
+The ```primary goal```of the Compliance Manager should be to achieve compliance. He is the advocate of the 3rd party license holders within the company. 
+
+Thus the role will be held ```accountable``` for 
+* Compliance of all open source used  within the products, that received his approval
+* Suitability of the general|all|project specific black/whitelists 
```

Comment:
Since up to now we were not using the term "black/whitelist" I would suggest to change this sentence to:
Suitability of the general|all|project specific policies

Reply:
I would not consider a black/whitelist equal to a policy. Also I am convinced that black/whitelist are an archaic method of managing OS licenses. However, they are still part of the daily work / instruments used out there. So we need someone being reponsible for managing and maintaining them. This I would put in the hands of this role.  If this makes sense to you, I could add the definition to our dictionary (do we have one?)

Reply:
I also think that working with black and white lists is a matter of the "stone age". I agree that this role shall be in charge of managing them. We do have a glossary:
Sharing-creates-value/Tooling-Landscape/Unanimous-Understanding/OSS-Tooling-Landscape-Glossary.md

Reply:
ok. added there.